virtwho/config.py

```diff
@@ -759,6 +764,11 @@ class ConfigSection(collections.MutableMapping):
     __marker = object()
 
     def __init__(self, section_name, wrapper):
+        """
+        Initialization of ConfigSection instance
+        :param section_name: The name of section
+        :param wrapper: ?
```

Comment:
@cnsnyder What exactly do you mean with `ConfigSection`'s parent? Is it parent class? In this case we can get it automatically using `self.__class__.__bases__`. Or do you mean other instance of ConfigSection (or it's subclass) referencing this ConfigSection (e.g. instance of LibVirtdConfigSection referencing to instance of VirtConfigSection?